af04f855-58c4-4db3-a191-45fe33381679/definition.xml

```diff
@@ -68,9 +68,11 @@
     <groupaction menu="Ready all my cards" execute="restoreAll" shortcut="F9" />
     <groupaction menu="Create Token" execute="createToken" shortcut="F10" />
     <groupaction menu="Create Synapse" execute="createSynapse" />
+    <groupaction menu="Create Army Unit" execute="createUnit" />
     <groupaction menu="End Turn" execute="endTurn" shortcut="F12" />
     <groupaction menu="STOP" execute="holdOn" shortcut="ALT+S" />
     <groupaction menu="I'm Done" execute="imDone" shortcut="ALT+D" />
+    <groupaction menu="Deck Check Tournament" execute="CheckTournamentValid" />
```

Comment:
Verify Deck Format seems clearer, I would imagine for casual play people might want to indicate the format as well